mod.json

```
@@ -1,17 +1,13 @@
 {
     "Name": "fvnkhead.killstat",
     "Description": "gather kill statistics",
-    "Version": "1.1.0",
+    "Version": "1.1.1",
```

This breaks the old format so:

```suggestion
    "Version": "2.0.0",
```